src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin: 20px auto;
+}
+
+.searchInput {
+  width: 100%;
+  font-family: Avenir, Helveica, sans-serif;
+  font-weight: 900;
+
+  color: #3d4e61;
+
+  border-color: #fff;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  font-size: 16px;
+  /* line-height: 22px; */
+  padding: 23px 20px 23px 62px;
+  background: url(./images/Search.svg) no-repeat 24px 24px/19px;
+}
+
+.searchInput::placeholder {
+  font-weight: 300;
+  font-family: inherit;
+  font-size: 16px;
+}
+
+.searchInput:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchInput:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
Magnifier icon dissappears on focus, you need to add it here
```suggestion
  background: url(./images/Search.svg) no-repeat 12px 14px/11px, 
  	linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Reply:
Please pay attention to my comments
![image](https://user-images.githubusercontent.com/37161315/137449464-80755aee-d077-463c-98e6-647a0a83e672.png)